lib/rubocop/cop/naming/predicate_name.rb

```diff
@@ -18,12 +18,35 @@ module Naming
       #   # good
       #   def value? ...
       class PredicateName < Cop
+        def_node_matcher :dynamic_method_define, <<-PATTERN
```

Comment:
This is a bit fragmented. Use don it's own, this doesn't return what it says it does. It just returns an unfiltered list of methods. This could probably use a funcall (see the top level comment in `NorePattern`) to filter according to the actual whitelist.

Reply:
I made a fix using funcall with the following commit 🙏 
31eca87